include/CppUTestExt/MockNamedValue.h

```diff
@@ -60,13 +60,41 @@ class MockFunctionComparator : public MockNamedValueComparator
 };
 
 /*
+ * MockNamedValueCopier is an interface that needs to be used when creating Copiers.
+ * This is needed when copying values of non-native type.
+ */
+
+class MockNamedValueCopier
+{
+public:
+    MockNamedValueCopier() {}
+    virtual ~MockNamedValueCopier() {}
+
+    virtual void copy(void* dst, const void* src)=0;
```

Comment:
For several reasons, all related:
1. `void *` represents a pointer to modifiable memory. `const void*` represents a pointer to non-modifiable memory. Of course, the object that I copy from shall be non-modifiable, but the object that I copy into shall be modifiable. Of course, I could declare `dst` as `const void*` then cast to non-const, but that would remove the "hint" in the interface definition that `dst` has to be modifiable and `src` not.
2. To mimic the "memcpy" interface.
3. Because the expected call method gets a `const void *`, and the actual call method gets a `const void*`, so it's pretty straightforward where each parameter finally goes into the call to the copier (so they're not messed the other way).